modules/ossm-remove-cleanup.adoc

```diff
@@ -0,0 +1,95 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/installing-ossm.adoc
+
+
+[id="ossm-remove-cleanup_{context}"]
+= Clean up Operator resources
+
+Follow this procedure to manually remove resources left behind after removing the {ProductName} Operator using the OpenShift console.
+
+.Prerequisites
+
+* An account with cluster administration access.
+* Access to the {product-title} Command-line Interface (CLI) also known as `oc`.
+
+.Procedure
+
+. Log in to the {product-title} CLI as a cluster administrator.
+
+. Run the following commands to clean up resources after uninstalling the Operators:
++
+[NOTE]
```

Comment:
The second note about Jaeger seems more like a Note. Do we really need to double note here? Can this sentence be a normal sentence?